Updated the tutorial (#2113)
vitessce/vitessce: sites/docs/docs/tutorial-gh-pages.mdx

git branch -M main
```

## Repo setup
Create a new GitHub repository (https://github.com/new) (here I called the repository `vitessce-demo-gh-pages`) and set it as the remote origin:

```sh
# Replace my-username with your GitHub username in the following line:
git remote add origin git@github.com:my-username/vitessce-demo-gh-pages.git
git push -u origin main

```

> ## Troubleshooting

Reviewer: ```suggestion
### Troubleshooting
```

 if you get error in pushing to main, try the following and commit the changes in the editor:

> ```

Reviewer: ```suggestion
```sh
```

>   git pull origin main --allow-unrelated-histories --no-rebase

Reviewer: ```suggestion
git pull origin main --allow-unrelated-histories --no-rebase
```

```

> 

Reviewer: ```suggestion
## Install Vitessce
```

> Install Vitessce:

Reviewer: ```suggestion
Install the Vitessce NPM package to be able to use the React component in the app:
```

}
```

### Configure Vite project

> Edit the `vite.config.js` file to allow Vite to serve correctly from a subpath like `https://yourname.github.io/your-repo-name/`

Reviewer: ```suggestion
Edit the `vite.config.js` file to allow Vite to serve correctly from a subpath like `https://my-username.github.io/vitessce-demo-gh-pages/`
```

      - name: Deploy to GitHub Pages
        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
        id: deployment
        uses: actions/deploy-pages@v4
    
```

> ## Config Vite

Reviewer: ```suggestion
## Configure Vite project
```

> Add the following to the `vite.config.js` file to allow Vite to serve correctly from a subpath like `https://yourname.github.io/your-repo-name/`

Reviewer: ```suggestion
Add the following to the `vite.config.js` file to allow Vite to build the website such that it can be served from a subpath like `https://my-username.github.io/vitessce-demo-gh-pages/`
```

> ```

Reviewer: ```suggestion
```js
```

export default defineConfig({
  plugins: [react()],
  base: '/vitessce-demo-gh-pages/',

>   # other config ...

Reviewer: ```suggestion
  // other config ...
```
